pkg/config/images.go

```diff
@@ -33,15 +32,10 @@ func ListK0sImages(cfg *k0sv1beta1.ClusterConfig) []string {
 		// skip these images
 		case cfg.Spec.Images.KubeRouter.CNI.URI(),
 			cfg.Spec.Images.KubeRouter.CNIInstaller.URI(),
-			cfg.Spec.Images.Konnectivity.URI():
+			cfg.Spec.Images.Konnectivity.URI(),
+			cfg.Spec.Images.PushGateway.URI():
```

Comment:
With https://github.com/k0sproject/k0s/pull/5520 push gateway now shows up on the airgap image list. My understanding is that we don't need/use this since it's used for metrics scrapping - https://docs.k0sproject.io/head/system-monitoring/ - that we don't enable. Based on that we can probably skip it altogether.